apps/builder/app/builder/features/topbar/entri.tsx

```diff
@@ -0,0 +1,153 @@
+import useScript from "react-script-hook";
+import type { DomainRouter } from "@webstudio-is/domain/index.server";
+import { createTrpcFetchProxy } from "~/shared/remix/trpc-remix-proxy";
+import { builderDomainsPath } from "~/shared/router-utils";
+import { Button, Text, globalCss } from "@webstudio-is/design-system";
+import { useEffect, useState } from "react";
+
+const trpc = createTrpcFetchProxy<DomainRouter>(builderDomainsPath);
+
+const scriptAttributes = {
+  async: true,
+  defer: true,
+  importance: "low",
+  src: "https://cdn.goentri.com/entri.js",
+};
+
+// https://developers.entri.com/docs/install
+type DnsRecord = {
+  type: "CNAME" | "TXT";
+  host: string;
+  value: string;
+  ttl: number;
+};
+
+// https://developers.entri.com/docs/integrate-with-dns-providers
+type EntriCloseDetail = {
+  domain: string;
+  lastStatus:
+    | "FINISHED_SUCCESSFULLY"
+    | "IN_PROGRESS"
+    | "EXISTING_RECORDS"
+    | "LOGIN"
+    | "MANUAL_CONFIGURATION"
+    | "EXIT_WITH_ERROR"
+    | "DKIM_SETUP";
+
+  provider: string;
+  setupType: "automatic" | "manual" | "sharedLogin" | null;
+  success: boolean;
+};
+declare global {
+  interface Window {
+    // https://developers.entri.com/docs/api-reference
+    entri: {
+      showEntri: (options: {
+        applicationId: string;
+        token: string;
+        dnsRecords: DnsRecord[];
+        prefilledDomain: string;
+      }) => void;
+    };
+  }
+
+  // https://developers.entri.com/docs/integrate-with-dns-providers
+  interface WindowEventMap {
+    onEntriClose: CustomEvent<EntriCloseDetail>;
+  }
+}
+
+/**
+ * Our FloatingPanelPopover adds pointerEvents: "none" to the body.
+ * We open the entry dialog from the popover, so we need to allow pointer events on the entri dialog.
+ */
+const globalStyles = globalCss({
+  body: {
+    "&>#entriApp": {
```

Comment:
instead In would just query that element when entri renders and set it inline and do that in our sentri wrapper, outside of react, less dep on specificity or radix

Reply:
Its 3rd party API there is no way to detect when entry element will come to DOM. It can be async etc. This is good way to workaround the issue

Reply:
wouldn't couple it with radix still, e.g. can use react's style element directly

Reply:
Don't get about radix, there is no radix here

Reply:
sorrry I meant stitches